### PR TITLE
Update classcloud.py

### PR DESCRIPTION
**Pull request by jerbaroo**
Untested change. Maybe you can test it Killian since you're actively working on this project.
I made the change in GitHub editor, havn't the project cloned locally.
@KillianDavitt

### CODE REVIEW

File: classcloud.py [hunk 1]
```diff
@@ -54,11 +54,10 @@ def list_files():
     return "No token", 400
   if data.get("token", None) != token:
     return "Invalid token", 400
-  files = [{file.relative_path() : file.id } for file in File.query.all()]
-  files = files[0]
-  #files.sort(key=lambda x: x[1]) # sort by full path
-  print(flask.jsonify(files=files))
-  return flask.jsonify(files), 200
+  file = File.query.first()
+  file = {file.relative_path(): file.id }
```
Comment (barischj) [inline classcloud.py]: Why no comment replies in GitHub 😢

**Comment by barischj** [comment]
but we can has conversation

File: classcloud.py (hunk 1 above)
Comment (barischj) [inline classcloud.py]: sort of a reply

**Comment by KillianDavitt** [comment]
This isn't correct, I had accidently left in 'files = files[0]' for some unknown reason.
The query is fetching all the files to list them, so it makes no sense to only take the first one.

My mistake for somehow leaving this in the code.

It's now fixed in the main branch,

files = Files.query.all()

File: classcloud.py [hunk 2]
```diff
@@ -54,11 +54,10 @@ def list_files():
     return "No token", 400
   if data.get("token", None) != token:
     return "Invalid token", 400
-  files = [{file.relative_path() : file.id } for file in File.query.all()]
-  files = files[0]
-  #files.sort(key=lambda x: x[1]) # sort by full path
```
Comment (jerbaroo) [inline classcloud.py:59]: Foo

Comment (jerbaroo) [reply classcloud.py:59]: Bar

Comment (jerbaroo) [reply classcloud.py:59]: @KillianDavitt real comment threads, and we only just complained about it

Comment (KillianDavitt) [reply classcloud.py:59]: This is excellent. Ta excite

**Pull request closed without merging**